test/csuite/random_directio/main.c

```diff
@@ -833,12 +813,7 @@ check_schema(WT_SESSION *session, uint64_t lastid, uint32_t threadid, uint32_t f
         if (LF_ISSET(SCHEMA_DATA_CHECK))
             check_one_entry(session, uri, uri, uri);
     }
-    if (LF_ISSET(SCHEMA_RENAME) && has_schema_operation(lastid, 2)) {
-        /* Table rename operation. */
-        gen_table_name(uri, sizeof(uri), lastid - 2, threadid);
-        gen_table2_name(uri2, sizeof(uri2), lastid - 2, threadid, flags);
-        if (LF_ISSET(SCHEMA_VERBOSE))
-            fprintf(stderr, " rename %s,%s\n", uri, uri2);
+    if (has_schema_operation(lastid, 2)) {
```

Comment:
I think you should delete new lines 816-821. The SCHEMA_DROP operation is handled in new lines 831-837. I think new lines 817-821 are for SCHEMA_RENAME combined with SCHEMA_DROP and/or SCHEMA_CREATE which are not needed without SCHEMA_RENAME.

Reply:
Yeah this makes sense, done here [657d423](https://github.com/wiredtiger/wiredtiger/pull/10306/commits/657d42348c32dbe6428c2101c1307ea781867e2e)